hr_holidays_compute_days/README.rst

```diff
@@ -0,0 +1,71 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Employee Compute Leave Days
+===========================
+
+Computes the actual days for which employee will be on leave taking into 
+account both rest days and public holidays
+
+Installation
+============
+
+No additional steps necessary
+
+Configuration
+=============
+
+Go to your holiday status and specify what leaves should not include rest days
```

Comment:
Go to your holiday status and specify for their computation which leaves should not include rest days
or holidays.